mozdownload/scraper.py

```diff
@@ -696,7 +696,14 @@ def get_build_info(self):
         self.show_matching_builds(parser.entries)
         self.builds = parser.entries
         self.build_index = len(parser.entries) - 1
-
+        
+        if self.build_number:
```

Comment:
You don't want to add this code here but integrate it in the condition right below.